wpm/game.py

```diff
@@ -234,14 +234,14 @@ def handle_key(self, key):
         # Browse mode
         if self.start is None or (self.start is not None and
                                   self.stop is not None):
-            if key in (" ", "KEY_LEFT", "KEY_RIGHT"):
+            if key in ("\t", "KEY_LEFT", "KEY_RIGHT"):
```

Comment:
Please put the tab in a separate PR if you think it should be there.

Reply:
Removed. The reason for having it was that I was often accidentally pressing spacebar right after the game reset due to a typo.

I agree that this should not be changed.